syntax = "proto3";
package webotsMotionEngine;


message degrees{
    repeated double motor_degs = 1;
    repeated uint32 motor_number = 2;
}

message sensorData{
    repeated double gyro = 1;
    repeated double accelerometer = 2;
}

message cameraData{
    uint32 height = 1;
    uint32 width = 2;
    string raw_data = 3;
    uint64 timeStep = 4;
}
